.gitignore

```diff
@@ -5,7 +5,8 @@ __pycache__/
 
 # C extensions
 *.so
-
+# tar.gz
+*.tar.bz2
```

Comment:
Let's do this in a separate PR, we should make each PR self-contained (also, when you do, please put it at the bottom of the file as that's where ignored files specific to this projects are).

Reply:
Done. :)